src/features/pastelDB/__tests__/pastelDBThread.test.ts

```diff
@@ -0,0 +1,473 @@
+          n: 0,
+          scriptPubkey: {
+            asm: '',
+            hex: '',
+            regSigs: 0,
+            type: '',
+            addresses: [''],
+          },
+        },
+      ],
+      vjoinsplit: [
+        {
+          vpub_old: 0,
+          vpub_new: 0,
+          anchor: '',
+          nullifiers: [''],
+          commitments: [''],
+          onetimePubKey: '',
+          randomSeed: '',
+          macs: [''],
+          proof: '',
+          ciphertexts: [''],
+        },
+      ],
+      blockhash: '',
+      confirmations: 0,
+      time: 0,
+      blocktime: 0,
+    }
+  })
+
+  const getRpcData = jest.fn(() => {
+    return {
+      height: 0,
+      bestblock: 'best-block-123456',
+      transactions: 0,
+      txouts: 0,
+      bytes_serialized: 0,
+      hash_serialized: '',
+      total_amount: 0,
+    }
+  })
+
+  const getChaintips = jest.fn(() => {
+    return {
+      height: 0,
+      hash: 'chain-tips-hash-1234578',
+      branchlen: 0,
+      status: '',
+    }
+  })
+
+  const getBlockSubSidy = jest.fn(() => {
+    return {
+      miner: 456789,
+      masternode: 0,
+      governance: 0,
+    }
+  })
+
+  const getWalletInfo = jest.fn(() => {
+    return {
+      walletversion: 456789,
+      balance: 0,
+      unconfirmed_balance: 0,
+      immature_balance: 0,
+      txcount: 0,
+      keypoololdest: 0,
+      keypoolsize: 0,
+      paytxfee: 0,
+      seedfp: '',
+    }
+  })
+
+  const listTransactions = jest.fn(() => {
+    return {
+      account: 'account-1',
+      address: '',
+      category: '',
+      amount: 0,
+      vout: 0,
+      confirmations: 0,
+      blockhash: 0,
+      blockindex: 0,
+      blocktime: 0,
+      expiryheight: 0,
+      txid: '',
+      walletconflicts: [''],
+      time: 0,
+      timereceived: 0,
+      vjoinsplit: [
+        {
+          vpub_old: 0,
+          vpub_new: 0,
+          anchor: '',
+          nullifiers: [''],
+          commitments: [''],
+          onetimePubKey: '',
+          randomSeed: '',
+          macs: [''],
+          proof: '',
+          ciphertexts: [''],
+        },
+      ],
+      size: 0,
+      lastblock: '',
+    }
+  })
+
+  const listUnspent = jest.fn(() => {
+    return {
+      txid: 'unspent-txid-123456789',
+      vout: 0,
+      generated: true,
+      address: '',
+      account: '',
+      scriptPubKey: '',
+      amount: 0,
+      confirmations: 0,
+      spendable: 0,
+    }
+  })
+
+  const getTotalBalance = jest.fn(() => {
+    return {
+      transparent: 'totalbalance-transparent',
+      private: '',
+      total: '',
+    }
+  })
+
+  const listAddresses = jest.fn(() => {
+    return 'address1'
+  })
+
+  test('statistic data should fetch and store correctly', async () => {
+    const info = getStatisticInfo()
+    insertStatisticDataToDB(pastelDB.db, info.hashrate, info.difficult)
+    const result = getDatafromDB('statisticinfo')
+    expect(result[0].values[0]).toContainEqual('1.2345')
+  })
```

Comment:
This is not right. Since database methods already tested in [pastelLib tests](https://github.com/pastelnetwork/pastel-electron-wallet/blob/42ba388af1f0a3fdf3d40165ef5498c2b98574a6/src/features/pastelDB/__tests__/pastelDBLib.test.ts), we can mock them because we can assume they work:

Here is how this test is supposed to look like:

```ts
import * as pastelDBLib from './pastelDBLib`
import * as netStats from '../../api/pastel-rpc/network-stats/network-stats' // after addressing all the comments, this should go away too

test('fetchStatisticinfo works', async () => {
  // Arrange
  const mockDB = { database: {} } as Database; // this is bad actually, but we don't have database interface at the moment. If we had, we could actually mock every field 
  const mockConfig = { username: 'test', password: 'pass' } // doesn't matter what's in there 
  const insertStatisticDataToDBSpy = jest.spyOn(pastelDBLib, 'insertStatisticDataToDBSpy')
  const getStatisticInfoSpy = jest.spyOn(netStats, 'getStatisticInfo') // this will be replaced with rpc method mock after my comments get addressed
  
  // Act
  const info = await fetchStatisticinfo({ pastelDB: mockDB, rpcConfig: mockConfig })

  // Assert
  expect(getStatisticInfoSpy).toHaveBeenCalled()
  expect(getStatisticInfoSpy).toHaveBeenCalledWith(mockConfig)

  expect(insertStatisticDataToDBSpy).toHaveBeenCalled()
  expect(insertStatisticDataToDBSpy).toHaveBeenCalledWith(mockDB, 10, 10)
})
```
Also please take a look at `Arrange`, `Act`, and `Assert` comments. They are good for visual separation of logic blocks. Please fix all your tests to look like this